content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Read replicas'
+metaTitle: 'Read replicas'
+metaDescription: 'This page explains how to set up and use read replicas with Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Read replicas enable you to distribute workloads across database replicas for high-traffic workloads. The [`@prisma/extension-read-replicas`](https://github.com/prisma/extension-read-replicas) extension adds read replication support to Prisma Client.
+
+The read replicas extension supports Prisma versions 5.2.0 and higher.
```

Comment:
meta comment. "read replicas extension" or "read replica extension"?

Reply:
"read replicas extension" — based on the name of the repo 🙂